### PR TITLE
Import images before rollout using helm (workaround for issue 1025)

### PR DESCRIPTION
**Pull request by quenteum**
fix for #1025

### CODE REVIEW

File: src/org/ods/orchestration/phases/DeployOdsComponent.groovy [hunk 1]
```diff
@@ -48,6 +48,9 @@ class DeployOdsComponent {
             if (!openShiftDir.startsWith('openshift')) {
                 deploymentDescriptor.deployments.each { String deploymentName, Map deployment ->
                     importImages(deployment, deploymentName, project.sourceProject)
+                }
+                deploymentDescriptor.deployments.each { String deploymentName, Map deployment ->
+                    // importImages(deployment, deploymentName, project.sourceProject)
```
Comment (jafarre-bi) [inline src/org/ods/orchestration/phases/DeployOdsComponent.groovy:53]: Can you please remove the commented code?

**Comment by jafarre-bi** [comment]
Can you please add a more understandable title to the PR?

**Review by jafarre-bi — CHANGES_REQUESTED** [review]

File: src/org/ods/orchestration/phases/DeployOdsComponent.groovy (hunk 1 above)
Comment (quenteum) [reply src/org/ods/orchestration/phases/DeployOdsComponent.groovy]: @jafarre-bi Done

**Comment by hrcornejo** [comment]
Please @quenteum update the CHANGELOG.md adding an entry in Unreleased part

**Review by hrcornejo — APPROVED** [review]

**Review by jafarre-bi — APPROVED** [review]

**Pull request merged**